client/signup/jetpack-connect/connect-header.jsx

```diff
@@ -1,40 +1,40 @@
+/**
+ * External Dependencies
+ */
 import React, { PropTypes } from 'react';
 
+/**
+ * Internal dependencies
+ */
 import StepHeader from '../step-header';
 
-export default React.createClass( {
-	displayName: 'JetpackConnectHeader',
+const JetpackConnectHeader = ( props ) => {
+	return (
+		<div className="jetpack-connect__header-container">
+			{ props.showLogo
+				? (
+					<img
+						className="jetpack-connect__jetpack-logo"
+						src="/calypso/images/jetpack/jetpack-logo.svg"
```

Comment:
hmm, did we merge it there? Yeah, I think so.. Anyway we should just remove this part (In fact probably this whole component), because we are not showing it anymore